reflex-host-examples.cabal

```diff
@@ -30,11 +30,12 @@ library
   build-depends:       base          >= 4.8  && < 4.10
                      , mtl           >= 2.2  && < 2.3
                      , transformers  >= 0.5  && < 0.6
-                     , dependent-sum >= 0.3  && < 0.4
+                     , dependent-sum >= 0.3  && < 0.5
                      , dependent-map >= 0.2  && < 0.3
                      , ref-tf        >= 0.4  && < 0.5
-                     , lens          >= 4.13 && < 4.15
+                     , lens          >= 4.13 && < 5.0
```

Comment:
As a note / neutral comment: I prefer to keep my bounds tied to versions that I've tested with, or in this case which reflect the bounds of the libraries that get pulled in by the work-on script.  Especially with think like `lens` where the second number is usually the stability number.

With that said, `reflex` uses the same upper bound for `lens` so it'll save me bumping some bounds until a `lens` change effects `reflex`, and the change works": so I'm happy and very grateful for the help :)

With that said, reflex uses a similar wide